pkg/proxy/proxy.go

```diff
@@ -269,10 +270,31 @@ func parseIPPort(ipstr string, info *accesslog.EndpointInfo) {
 	}
 }
 
-func (r *Redirect) getSourceInfo(req *http.Request) (accesslog.EndpointInfo, accesslog.IPVersion) {
+/*
```

Comment:
It is common to have a multiline comment with `//`. For example https://golang.org/src/builtin/builtin.go?#L125

Reply:
sure.